.eslintrc.js

```diff
@@ -3,19 +3,19 @@
 module.exports = {
     extends: [
         "react-app",
-        "prettier",
         "eslint:recommended",
         "plugin:react/recommended",
         "plugin:cypress/recommended",
         "plugin:@typescript-eslint/recommended",
-        "prettier/@typescript-eslint",
-        "plugin:prettier/recommended",
     ],
     rules: {
```

Comment:
When I run "yarn link" I get 5 errors about rule "no-redeclare" . Are you seeing those?

Reply:
Fixed, the cypress configuration files had redeclaration of global variables as the cy object... I've already had to remove those from other projects in the past (training-app for example).